src/main/webapp/WEB-INF/tags/instructor/feedbacks/feedbackSessionActions.tag

```diff
@@ -0,0 +1,100 @@
+<%@ tag description="instructorFeedbacks and instructorHome - Feedback Session actions" %>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
+<%@ taglib tagdir="/WEB-INF/tags/instructor/feedbacks" prefix="tif" %>
+<%@ tag import="teammates.common.util.Const" %>
+<%@ attribute name="actions" type="teammates.ui.template.FeedbackSessionActions" required="true" %>
+<c:if test="${actions.hasActions}">
+    <a class="btn btn-default btn-xs btn-tm-actions session-view-for-test"
+       href="${actions.resultsLink}"
+       title="<%= Const.Tooltips.FEEDBACK_SESSION_RESULTS %>"
+       data-toggle="tooltip"
+       data-placement="top">
+        View Results
+    </a>
+    <a class="btn btn-default btn-xs btn-tm-actions session-edit-for-test"
+       href="${actions.editLink}" 
+       title="<%= Const.Tooltips.FEEDBACK_SESSION_EDIT %>"
+       data-toggle="tooltip"
+       data-placement="top"
+       <c:if test="${not actions.allowedToEdit}">disabled="disabled"</c:if>>
+        Edit
+    </a>
+    <a class="btn btn-default btn-xs btn-tm-actions session-delete-for-test"
+       href="${actions.deleteLink}"
+       title="<%= Const.Tooltips.FEEDBACK_SESSION_DELETE %>"
+       data-toggle="tooltip"
+       data-placement="top"
+       onclick="return toggleDeleteFeedbackSessionConfirmation(${actions.toggleDeleteFeedbackSessionParams});"
+       <c:if test="${not actions.allowedToDelete}">disabled="disabled"</c:if>>
+        Delete
+    </a>
+    <a class="btn btn-default btn-xs btn-tm-actions session-copy-for-test"
+       href="#" 
+       title="<%= Const.Tooltips.FEEDBACK_SESSION_COPY %>"
+       data-actionlink="${actions.editCopyLink}"
+       data-courseid="${actions.courseId}"
+       data-fsname="${actions.fsName}"
+       data-toggle="modal"
+       data-target="#fsCopyModal"
+       data-placement="top"
+       id="button_fscopy-${actions.courseId}-${actions.fsName}">
+        Copy
+    </a>
+    <div title="<%= Const.Tooltips.FEEDBACK_SESSION_SUBMIT %>"
+         data-toggle="tooltip"
+         data-placement="top"
+         style="display: inline-block; padding-right: 5px;">
+        <a class="btn btn-default btn-xs btn-tm-actions session-submit-for-test<c:if test="${not actions.hasSubmit}"> disabled</c:if>"
+           href="${actions.submitLink}"
+           <c:if test="${not actions.hasSubmit}">onclick="return false"</c:if>
+           <c:if test="${not actions.allowedToSubmit}">disabled="disabled"</c:if>>
```

Comment:
Here too, seems like there's alot of this, I wonder how many more of these there are and whether we are using .prop() or .removeClass() / .addClass() when dealing with these in JavaScript?